### PR TITLE
Update main.html

### PR DESCRIPTION
**Pull request by potatoyenn**

### CODE REVIEW

File: app_server/views/main.html [hunk 1]
```diff
@@ -26,75 +34,83 @@
...
         <ul id="subNavList">
             <li id="eventsMenu" ><a href="#">Events</a></li>
             <li id="mealMenu"><a href="#">Meal Share</a></li>
-            <li id="goodsMenu"><a href="#">Goods</a></li>
+            <li id="goodsMenu"><a href="#">Goods Exchange</a></li>
             <li id="petsMenu"><a href="#">Pet Services</a></li>
             <li id="ridesMenu"><a href="#">Ride Share</a></li>
         </ul>
     </div>
-    </div>
```
Comment (potatoyenn) [inline app_server/views/main.html:81]: mistake

File: app_server/views/main.html [hunk 2]
```diff
@@ -497,27 +531,24 @@
...
 
 <div id="rightMap" class="">
-    <div id="mapLocationTitle" class="col-md-12"><strong>My Location</strong></div>
+    <div id="mapLocationTitle" class="col-md-12">
+        <strong>My Location</strong>
+    </div>
     <img id="map" src="map.png">
 </div>
-
-</div>
```
Comment (potatoyenn) [inline app_server/views/main.html:517]: mistake

File: app_server/views/main.html [hunk 3]
```diff
@@ -1,20 +1,28 @@
...
+    
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Satisfy" >
     <link rel="stylesheet" type="text/css" href="mainStyle.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    
+    //The 3 below were originally at the bottom of the doc
+    <script src="jquery-3.1.1.min.js"></script>
```
Comment (potatoyenn) [inline app_server/views/main.html:17]: place back on bottom

File: app_server/views/main.html [hunk 4]
```diff
@@ -1,20 +1,28 @@
...
+    <script src="mainJS.js" type="text/javascript"></script>
 </head>
 
 <body class="bodyMain">
-
-<!--Main navigation bar begins here-->
-
+    
+<!--Beginning of Main Navigation Bar-->   
+//remove the negative margins by adding no-gutter
```
Comment (potatoyenn) [inline app_server/views/main.html:25]: no // for comments; will fix padding later

**Review by RobertoAF — APPROVED** [review]
